tfjs-core/src/io/indexed_db_test.ts

```diff
@@ -172,7 +172,7 @@ describeWithFlags('IndexedDB', BROWSER_ENVS, () => {
 
        try {
          await handler.load();
-         fail('Loading nonexistent model from IndexedDB succeeded unexpectly');
+         fail('Loading nonexistent model from IndexedDB succeeded unexpectedly');
```

Comment:
Looks like this line is too long:
```
/workspace/tfjs-core/src/io/indexed_db_test.ts:175:1
ERROR: 175:1  max-line-length  Exceeds maximum line length of 80
```

```suggestion
         fail('Loading nonexistent model from IndexedDB succeeded'
               + ' unexpectedly');
```